draft-ietf-cose-merkle-tree-proofs.md

```diff
@@ -160,21 +160,20 @@ while switching from one verifiable data structure to another, so long as both s
 This document establishes a registry of verifiable data structure proof types tags,
 with the following initial contents:
 
-| Identifier  | Proof Type   | Reference
+| Identifier  | Proof Type   | Proof Value | Reference
 |---
-|0            | N/A          |
-|TBD_2        | inclusion    | {{sec-generic-inclusion-proof}}
-|TBD_3        | consistency  | {{sec-generic-consistency-proof}}
+|0            | N/A          | N/A | N/A
+|TBD_2        | inclusion    | bstr or list of bstr | {{sec-generic-inclusion-proof}}
+|TBD_3        | consistency  | bstr or list of bstr | {{sec-generic-consistency-proof}}
 {: #verifiable-data-structure-proof-types-values align="left" title="Verifiable Data Structure Proof Types"}
 
-Editors note: The registry requirements needs to address the case of multiple proofs of a given type.
-
 ## Inclusion Proof {#sec-generic-inclusion-proof}
 
 Inclusion proofs provide a mechanism for a verifier to validate set membership.
 
 The integer identifier for this Proof Type is TBD_2.
 The string identifier for this Proof Type is "inclusion".
+The value of this Proof Type is bstr or list of bstr.
```

Comment:
Wouldn't it be better/simpler for this to ALWAYS be a list and sometimes only have 1 element?